docs/scos/dev/glue-api-guides/202204.0/glue-backend-api/how-to-guides/how-to-use-authentication-server.md

```diff
@@ -0,0 +1,132 @@
+---
+title: How to use authentication server
+description: This document describes how to use an authentication server.
+last_updated: October 24, 2022
+template: howto-guide-template
+redirect_from:
+  - /docs/scos/dev/glue-api-guides/202204.0/glue-backend-api/how-to-guides/using-authentication-server.html
+---
+
+This document describes how to use an authentication server.
+
+Integrate authentication following the [Glue API Authentication integration](/docs/scos/dev/technical-enhancement-integration-guides/glue-authentication-integration.html) guide.
+
+Glue allows switching between different authentication servers. In order to provide it the `AuthenticationServerPluginInterface` must be implemented.
```

Comment:
Please add some additional text that explains why do we need this server plugins